src/lib/components/aggregate-fiat-estimate/aggregate-fiat-estimate.svelte

```diff
@@ -29,29 +30,36 @@
     }
   }
 
-  $: priceStore = fiatEstimates.price(tokenAddresses ?? []);
+  export let prices: Prices | undefined = undefined;
```

Comment:
```suggestion
  /**
    Pass prices if they've been fetched externally already (e.g. in a load function).
    If undefined, it will call in and wait for prices from `fiatEstimates` store.
    
    If you do pass prices, make sure that there's a value for each token address included in `amounts`.
  */
  export let prices: Prices | undefined = undefined;
```

Let's just add a jsdoc comment here for clarity, since this isn't a common pattern